source/request-as-event-emitter.js

```diff
@@ -142,13 +142,7 @@ module.exports = (options = {}) => {
 
 		if (backoff) {
 			retryCount++;
-			setTimeout(options => {
-				try {
-					get(options);
-				} catch (error2) {
-					emitter.emit('error', error2);
-				}
-			}, backoff, options);
```

Comment:
Why was this removed?

Reply:
`get()` does not throw any errors directly. It uses `emitter.emit('error', error);`

Reply:
Good point. I think we need a try/catch here though: https://github.com/sindresorhus/got/pull/542/files#diff-62bdc57f6f22ae58f495daef16f21f8bR141 In case the `options.gotRetry.retries` function throws.

Reply:
Your answer is not confusing, but you're not answering my question. Too be clear, I think we need to try/catch this line because it could throw: `const backoff = options.gotRetry.retries(++retryTries, error);`

Reply:
> you're not answering my question.

Sorry, if you're talking about the second one then it hadn't popped up when I was writing the answer. Pardon my confusion.

> I think we need to try/catch this line because it could throw

Good point! Indeed. 🙌

Reply:
> I think we need to try/catch this line because it could throw

Reply:
>> I think we need to try/catch this line because it could throw
> Good point! Indeed. 🙌

And of course a test ;)

Reply:
Done! I hope it satisfies you :)